<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layerConfigurationDialog</class>
 <widget class="QDialog" name="layerConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Layer Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="watcherResources.qrc">
    <normaloff>:/icons/resources/watcher.png</normaloff>:/icons/resources/watcher.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="layerActiveCheckBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
  </widget>
  <widget class="QListWidget" name="layerListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>491</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="edgeGroupBox">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>1</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Edge Settings</string>
   </property>
   <widget class="QLabel" name="edgeLabelColorLabel">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>31</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Label Color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="edgeLabelColorButton">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>31</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgeColorLabel">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>100</y>
      <width>35</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="edgeColorButton">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>100</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="edgeFlashingCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>175</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Flash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgeLabelFontButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Font</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>206</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge Flash Rate</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="edgeFlashingRateSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>206</y>
      <width>129</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="edgeWidthSpinBox">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>143</y>
      <width>129</width>
      <height>26</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>143</y>
      <width>39</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgeLabelEditBox">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>68</y>
      <width>129</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>68</y>
      <width>36</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="nodeLabelGroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Attached Label Settings</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>114</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Foreground Color</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="edgeFlashingCheckBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>56</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Node Flash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nodeLabelFontButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Font</string>
    </property>
   </widget>
   <widget class="QToolButton" name="nodeLabelColorButton">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>25</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="floatingLabelGroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Floating Label Settings</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>114</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Foreground Color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="floatingLabelColorButton">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>25</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="edgeFlashingCheckBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>56</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floatingLabelFontButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Font</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="watcherResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setCurrentLayer(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edgeLabelEditBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>updateEdgeLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeLabelColorButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setNodeLabelColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edgeColorButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setEdgeColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>floatingLabelColorButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setFloatingLabelColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edgeLabelColorButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setEdgeLabelColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edgeWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setEdgeWidth(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edgeLabelFontButton</sender>
   <signal>clicked()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setEdgeLabelFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>floatingLabelFontButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setFloatingLabelFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeLabelFontButton</sender>
   <signal>released()</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>setNodeLabelFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerActiveCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>layerConfigurationDialog</receiver>
   <slot>layerToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCurrentLayer(QListWidgetItem*)</slot>
  <slot>setEdgeLabelFont()</slot>
  <slot>setEdgeLabelColor()</slot>
  <slot>setFloatingLabelColor()</slot>
  <slot>setNodeLabelColor()</slot>
  <slot>setEdgeWidth(int)</slot>
  <slot>updateEdgeLabel(QString)</slot>
  <slot>setEdgeColor()</slot>
  <slot>setNodeLabelFont()</slot>
  <slot>setFloatingLabelFont()</slot>
  <slot>layerToggled(bool)</slot>
 </slots>
</ui>
